head.go

```diff
@@ -771,14 +771,14 @@ func (h *headIndexReader) Close() error {
 	return nil
 }
 
-func (h *headIndexReader) Symbols() (map[string]struct{}, error) {
+func (h *headIndexReader) Symbols() (map[string]int, error) {
 	h.head.symMtx.RLock()
 	defer h.head.symMtx.RUnlock()
 
-	res := make(map[string]struct{}, len(h.head.symbols))
+	res := make(map[string]int, len(h.head.symbols))
 
 	for s := range h.head.symbols {
-		res[s] = struct{}{}
+		res[s] = 0
```

Comment:
Why is this `0`? It should be:
```
for s, num := range h.head.symbols {
    res[s] = num
}
```

Reply:
If I had a reason, I don't remember at this point :)